blog/secadvisor.md

```diff
@@ -0,0 +1,290 @@
+---
+title: How to use Security Advisor with Skydive
+layout: blog-post
+author: Kalman Meth
+date: 31/07/2019
+---
+
+
+The Security Advisor filters the flow data obtained from Skydive, performs a data transformation, and saves the information to an object store in compressed JSON format.
```

Comment:
in compressed JSON --> in GZIP compressed JSON encoding format.

Reply:
OK